TerrorConsole/Assets/_Main/Scripts/Shadow/CapturePlayer.cs

```diff
@@ -1,11 +1,14 @@
 using System;
 using UnityEngine;
+using UnityEngine.Events;
 
 namespace TerrorConsole
 {
     public class CapturePlayer : MonoBehaviour
     {
        [SerializeField] Collider2D ShadowCollider;
+       public UnityEvent OnplayerCaptured;
+       public UnityEvent OnplayerRespawn;
```

Comment:
These events were mant to be on the GameManager or LevelManager, something global, not part of the enemy haha